src/Concerns/Testable.php

```diff
@@ -73,6 +73,7 @@ public function __construct(Closure $test, string $description, array $data)
     {
         $this->__test        = $test;
         $this->__description = $description;
+        self::$beforeAll     = self::$afterAll = null;
```

Comment:
I know this is ripe coming from the guy who promotes higher order expectations inside higher order tests but should we split this over two lines for clarity?

Reply:
Probably, yeah. It is kind of easy to miss, if not careful. Two lines it is!

Reply:
Updated!